openproblems/data/utils.py

```diff
@@ -48,6 +48,7 @@ def loader(func, *args, **kwargs):
         log.debug("Downloading {}({}, {}) dataset".format(func.__name__, args, kwargs))
         adata = func(*args, **kwargs)
         adata.__from_cache__ = False
+        adata.layers["counts"] = adata.X
```

Comment:
this would at minimum require a check if `adata.layers['counts']` is already set I think (as is the case for the batch integration PR.

Reply:
This seems reasonable